reana_client/cli/workflow.py

```diff
@@ -856,13 +893,15 @@ def workflow_open_interactive_session(ctx, workflow, interactive_session_type,
                     fg="red", err=True)
 
 
-@interactive_group.command(
-    'close',
-    help='Close an interactive workflow session')
+@interactive_group.command('close')
 @add_workflow_option
 @add_access_token_options
-def workflow_close_interactive_session(workflow, access_token):
-    """Close an interactive workflow session."""
+def workflow_close_interactive_session(workflow, access_token):  # noqa: D301
+    """Close an interactive workflow session.
+
```

Comment:
Add some body, e.g.

The `close` command permits to shut down any interactive sessions that you may have running.  You would typically use this command after you finished exploring data in the Jupyter notebook and after you have transferred any code created in your interactive session.